src/app/components/courseInfoModal/CourseInfoModal.tsx

```diff
@@ -38,21 +49,13 @@ export default function CourseInfoModal() {
     },
     skip: course === undefined || isCustomCourse(course),
   });
-  const [customTitle, setCustomTitle] = useState("");
 
-  const handleEndEditing = () => {
-    if (course && editing && customTitle.length > 0) {
-      setEditing(false);
-      course.title = customTitle;
-      editCustomCourse(course.id, customTitle);
-    }
-  };
-
-  const handleTitleChange = (e: ChangeEvent<HTMLInputElement>) => {
-    const value = e.target.value;
-    setCustomTitle(value);
-  };
+  // This is to prevent accidentally opening the modal when it shouldn't be
+  if (course === undefined || course.departmentCode === undefined) {
```

Comment:
move this into a function called something like `function preventIllegalModalOpening()`